### PR TITLE
Makes fire priority values defines

### PR DESCRIPTION
**Pull request by ninjanomnom**
...again, except keeping the original values

### CODE REVIEW

**Comment by Jordie0608** [comment]
So many defines why even have them.

File: code/controllers/subsystem/processing/projectiles.dm [hunk 1]
```diff
@@ -1,6 +1,5 @@
 PROCESSING_SUBSYSTEM_DEF(projectiles)
 	name = "Projectiles"
-	priority = 25
```
Comment (Jordie0608) [inline code/controllers/subsystem/processing/projectiles.dm]: This one hasn't been replaced.

File: code/controllers/subsystem/radiation.dm [hunk 2]
```diff
@@ -1,7 +1,6 @@
 PROCESSING_SUBSYSTEM_DEF(radiation)
 	name = "Radiation"
 	flags = SS_NO_INIT | SS_BACKGROUND
-	priority = 25
```
Comment (Jordie0608) [inline code/controllers/subsystem/radiation.dm]: Nor this.

**Review by Jordie0608 — DISMISSED** [review]

**Comment by ninjanomnom** [comment]
So it's easier to tell what subsystems have higher priority than others by looking at a single place in code rather than having to go look at each subsystem individualy

File: code/controllers/subsystem/processing/projectiles.dm (hunk 1 above)
Comment (ninjanomnom) [reply code/controllers/subsystem/processing/projectiles.dm]: These are both the same as the parent

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Seperate SS_BACKGROUND subsystems into a different list
```
[03:55:17] <%MrStonedOne> background subsystems have their own priority bucket
```

**Comment by ninjanomnom** [comment]
@Cyberboss feels like a pointless change imo and a lot of background subsystems just use the default anyway. Easier to just lookup based on subsystem name than also have know whether it's a background subsystem or not to know which section to look in.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**